pyproject.toml

```diff
@@ -91,7 +91,7 @@ rocketchat_API = ">=0.6.31,<1.4.0"
 colorhash = "~1.0.2"
 pika = "~1.1.0"
 jsonschema = "~3.2"
-packaging = "~19.0"
+packaging = ">=20.0,<21.0"
```

Comment:
[changelog](https://github.com/pypa/packaging/blob/master/CHANGELOG.rst#203---2020-03-05) seems OK

Reply:
is there a reason not to support version 19?

Reply:
yes I need at least `20.0` because I rely on this:
> Add major, minor, and micro aliases to packaging.version.Version

Reply:
this requirement also prevents to update Rasa X release script because it will require an updated version of `rasa` otherwise the versions of `packaging` will conflict. Since we do not release alpha or release candidates yet in Rasa X, I suggest we update the script later (I can open an issue for that). WDYT?